uint8 opcode 
uint16 endpoint_id
uint8 reserved
uint32 value
float32 float_value